src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -41,146 +37,107 @@ class RuleDetails extends Component {
-                                        <div className='pf-u-display-inline-flex'>
-                                            <Shield
-                                                hasTooltip={ false }
-                                                impact={ this.ruleResolutionRisk(rule) }
-                                                size={ 'md' }
-                                                title={ AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
-                                            />
-                                        </div>
-                                        <div className='pf-u-display-inline-flex'>
-                                            <span className={ `label pf-u-pl-sm ins-sev-clr-${this.ruleResolutionRisk(rule)}` }>
-                                                { AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
-                                            </span>
-                                        </div>
-                                    </div>
-                                    <p>{ AppConstants.RISK_OF_CHANGE_DESC[this.ruleResolutionRisk(rule)] }</p>
-                                </GridItem>
-                                { this.ruleRebootRequired && (<GridItem>
-                                    <span className='ins-sev-clr-4'>Restart Required</span>
-                                </GridItem>) }
-                            </Grid>
+                        <GridItem sm={ 4 } md={ 12 } className='pf-l-flex'>
+                            <div>Risk of Change</div>
+                            <div className='pf-u-display-inline-flex pf-m-align-center pf-u-pb-sm pf-u-pt-sm'>
+                                <div className='pf-u-display-inline-flex'>
+                                    <Shield
+                                        hasTooltip={ false }
+                                        impact={ this.ruleResolutionRisk(rule) }
+                                        size={ 'md' }
+                                        title={ AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
+                                    />
+                                </div>
+                                <div className='pf-u-display-inline-flex'>
+                                    <span className={ `label pf-u-pl-sm ins-sev-clr-${this.ruleResolutionRisk(rule)}` }>
+                                        { AppConstants.RISK_OF_CHANGE_LABEL[this.ruleResolutionRisk(rule)] || 'Undefined' }
+                                    </span>
+                                </div>
+                            </div>
+                            <p>{ AppConstants.RISK_OF_CHANGE_DESC[this.ruleResolutionRisk(rule)] }</p>
                         </GridItem>
+                        { rule.reboot_required && (<GridItem>
```

Comment:
`reboot_required` is resolution specific per a discussion with @PaulWay.  Confirmed when comparing against master.

Reply:
check whats in prod yo... @PaulWay this is what i see for a rule...
```
active: true
category: {id: 3, name: "Stability"}
created_at: "2019-02-07T14:02:34.744417-05:00"
description: "Performance degradation of I/O when commands timeout due to faulty storage hardware"
generic: ""Abort Command Issued" messages are being detected, which are indicative of a storage area network (SAN) or hardware error.↵↵Verify if there are any issues present from the FC switch, FC cabling, zoning, or storage array. Red Hat recommends that you contact the storage vendor to review the issue encountered."
has_playbook: false
id: 80
impact: {name: "Storage Performance Loss", impact: 2}
impacted_systems_count: 0
likelihood: 2
more_info: ""
node_id: "27624"
publish_date: "2016-10-31T04:08:30-04:00"
reason: "This host has encountered **{{=pydata.total_aborts}}** occurrences of *Abort command issued* in /var/log/messages, including {{?pydata.abort_groups > 1}}multiple occurrences{{??}}1 occurrence{{?}} of 10 or more abort commands occurring within a period of an hour.  ↵↵*Abort command issued* messages indicate an error condition being returned from the SAN.↵↵An example abort command that is displayed in /var/log/messages: ↵↵**{{=pydata.example_abort}}**"
reboot_required: false
reports_acked: false
resolution_set: [{system_type: 105,…}]
rule_id: "abort_command_issued|ABORT_COMMAND_ISSUED"
ruleset: {created_at: "2018-05-21T22:00:51-04:00", updated_at: "2018-05-21T22:00:51-04:00",…}
summary: "Occurrences of the message "Abort Command Issued"  indicate an error condition being returned from the storage area network (SAN)."
tags: "sbr_kernel kernel storage"
total_risk: 2
updated_at: "2019-02-11T15:21:37.175020-05:00"
```

Reply:
seeeeee `reboot_require`?

Reply:
As long as we're getting the correct end result I'm g2g.  What I'm seeing is different rules being flagged as requiring Reboots depending on if you're querying `rule.reboot_required` or checking the resolution's value.